docs/trusted-resources.md

```diff
@@ -91,3 +91,36 @@ metadata:
 data:
   publickeys: "/etc/verification-secrets/cosign.pub, /etc/verification-secrets/cosign2.pub"
 ```
+
+#### Config key at VerificationPolicy
+VerificationPolicy supports SecretRef or encoded public key data.
+
+**Note:** Trusted Resources will first look up `VerificationPolicy` in the taskrun/pipelinerun's namespace, so the namespace should be the same of corresponding resources' namespace. If no `VerificationPolicy` it will check the system namespace, by default it is `tekton-pipelines`. If still no policy is found, it will try to fetch keys from configmap as described in last section.
```

Comment:
`it will try to fetch keys from configmap as described in last section.` <-- could you link to the section instead? e.g. `[it will try to fetch keys from configmap](#config-key-at-configmap).`

Also: Why do we want to support both specifying the key via the ConfigMap AND via the VerificationPolicy? Or will the ConfigMap support be removed in a later PR? It seems like maybe it's just adding unnecessary to have two different ways of specifying this?

Reply:
yes you're right! The current configmap is a simple solution to fetch keys. We should consider to remove it and only use `VerificationPolicy` once we finish the needed features of `VerificationPolicy` 

I will update the refer link as well.